internal/service/ch/queries.go

```diff
@@ -343,3 +343,15 @@ func getFilterMods(filter *model.SignalFilter) []qm.QueryMod {
 	}
 	return mods
 }
+
+// getDeviceActivityQuery returns the most recent 3 hour bucket during which device activity was recorded.
+func getDeviceActivityQuery(vehicleTokenID int, adManuf string) (string, []any) {
+	mods := []qm.QueryMod{
+		selectInterval(3 * time.Hour.Milliseconds()),
```

Comment:
Neat! I didn't know duration had functions for converting to other times. I have always done 
```go
time.Hour * time.Milliseconds
```